examples/aspnetcore/AspNetCore.CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+﻿{

```

Comment:
Can this be launched on macOS? (just curious)

Reply:
yes, `dotnet run` this proj, then browser `http://localhost:5000/api/TestScope`